redis.conf

```diff
@@ -173,7 +173,7 @@ timeout 0
 # On other kernels the period depends on the kernel configuration.
 #
 # A reasonable value for this option is 300 seconds, which is the new
-# Redis default starting with Redis 3.2.1.
```

Comment:
There is no Valkey 3.2.1 version, I prefer to keep the Redis 3.2.1 or indicate it origins from Redis 3.2.1

Reply:
We discussed this in a core team meeting. Let's just remove the references to old Redis versions. (At least we can remove them *in most cases*.)